pyproject.toml

```diff
@@ -19,15 +19,14 @@ requires-python = ">=3.10"
 dynamic = ["version"]
 
 dependencies = [
-    "mqt.bench>=1.1.1, <1.2.0",
+    "mqt.bench @ git+https://github.com/cda-tum/mqt-bench.git@main",
+    #"mqt.bench>=1.1.1, <1.2.0",
     "pytket_qiskit>=0.52.0, <0.54.0",
     "sb3_contrib>=2.0.0, <2.3.1",
     "scikit-learn>=1.4.0,<1.4.3",
-    "importlib_metadata>=4.4; python_version < '3.10'",
-    "importlib_resources>=5.0; python_version < '3.10'",
     "tensorboard>=2.11.0, <2.16.3",
     "bqskit>=1.1.2, <1.2.0",
-    "lark>=0.7, <1.0"
+    "lark~=0.7"
```

Comment:
Why the change here again? These two are equivalent and either will be updated by dependabot.